db/schema.sql

```diff
@@ -0,0 +1,29 @@
+/*database: rottentomatoes*/
+
+CREATE TABLE theaters (
+  ID SERIAL PRIMARY KEY,
+  name varchar(80) NOT NULL UNIQUE,
+  address varchar(80),
+  city varchar(80),
+  state varchar(20),
+  zip bigint,
+  phone varchar(20)
+)
+
+CREATE TABLE movies (
+  ID SERIAL PRIMARY KEY,
+  tmdb_id varchar(20) NOT NULL UNIQUE,
+  title varchar(80) NOT NULL UNIQUE,
+  title_url varchar(80),
+  tmdb_poster_path varchar(80),
+  tmdb_backdrop_path varchar(80)
+)
+
+CREATE TABLE showtimes (
+  ID SERIAL PRIMARY KEY,
+  week_day integer,
+  start_time varchar(80),
```

Comment:
Could consider TIME as a datatype

Reply:
I think the reason I chose integer is because 

> new Date().getDay()

 will return 0-6 and I use it in the client side code
I am cycling through the showtimes based on week day instead of real dates, but yes, in real world, I probably wouldn't use week day to store time at all.